### ISSUE TITLE
Compilation Problem

### ISSUE THREAD

**Issue by RobinLi-95** [zoowe/tpot#1]
Hi,

Thanks a lot for sharing the codes for the constant Fermi-level method implemented within VASP. 

I tried to modified the VASP code and compiled (vasp.5.4.4 + VASPSOL) that but met with the following error message:

```
pot.F(63): error #6401: The attributes of this name conflict with those made accessible by a USE statement.   [COMM_INTER]
      TYPE (communic)    COMM_INTER
-------------------------^
pot.F(403): error #6405: The same named entity from different modules and/or program units cannot be referenced.   [COMM_INTER]
         CALL M_bcast_d(COMM_INTER, SV(1,ISP), GRID%RL%NP)
------------------------^
```
I used the mpi/intel-2019 and intel-suite/2019.4 for compiling. Could you please help with this problem?

Best Regards

**Comment by zoowe** [comment]
It does not look like that this error relates to VASPSol or TPOT at all. Are you able to compile without VASPSol and TPOT?
I have used many versions of intel compilers and also with gnu compiler without a similar problem,

**Comment by RobinLi-95** [comment]
> It does not look like that this error relates to VASPSol or TPOT at all. Are you able to compile without VASPSol and TPOT? I have used many versions of intel compilers and also with gnu compiler without a similar problem,

Hi,

Thanks a lot for your reply.

I have successfully compiled VASP with VASPsol with the version without TPOT (and without modification of the VASP code). I found the aforementioned error after modifying the pot.F file following the guide.

**Comment by zoowe** [comment]
Please send me an email. I was not able to reproduce the issue.

**Comment by zoowe** [comment]
Resolved #[acc009e](https://github.com/zoowe/tpot/commit/acc009e2b89cd5b5b8281550ec135fbf377074a1)
